include/fiction/algorithms/physical_design/hexagonalization.hpp

```diff
@@ -362,7 +363,10 @@ template <typename HexLyt, typename CartLyt>
         }
     }
 
-    return offset;
+    auto offset_is_negative = offset < 0;
+    auto abs_offset         = std::abs(offset);
+
+    return std::make_pair(abs_offset, offset_is_negative);
```

Comment:
You're already computing the offset here, yet in the places where this result is used, the `offset_is_negative` flag is again checked, and another sign inversion is conducted. This seems excessive. Can this function not already handle all such cases?